templates/commands/render/render_action_include.go

```diff
@@ -68,6 +66,23 @@ func includePath(ctx context.Context, inc *spec.IncludePath, sp *stepParams) err
 			return err
 		}
 
+		// By default, we copy from the template directory. We also support
+		// grabbing files from the destination directory, so we can modify files
+		// that already exist in the destination.
+		fromDir := sp.templateDir
+		if inc.From.Val == "destination" {
+			fromDir = sp.flags.Dest
+		}
+
+		absSrcPaths, err := filepath.Glob(filepath.Join(fromDir, walkRelPath))
```

Comment:
Do we need to clean the result of the join with `filepath.Clean`? I think this would theoretically allow someone to execute a template outside of their PWD - is that desired?

Reply:
We're already protected from this by the call to `safeRelPath` above.